src/parse-stream.js

```diff
@@ -412,4 +417,28 @@ export default class ParseStream extends Stream {
       data: line.slice(4)
     });
   }
+
+  /**
+   * Add a parser for custom headers
+   *
+   * @param {RegExp}   expresion  a regular expression to match the custom header
+   * @param {String}   type       the custom type to register to the output
```

Comment:
I think this should be either `customType` or the param below can be renamed to `type`